### DISCUSSION TITLE
DEAR iamgui logo

### DISCUSSION THREAD

**Discussion by ghost** [ocornut/imgui#6676] [New Users Build/Link/Run issues ONLY!]
Just a little here and there for finding "dear-imgui " logo

the previous content of this message is not correct so deleted this

**Comment by ocornut** [comment]
What do you need a logo for?

Please don’t use this logo you posted. It comes from a silly logo contest someone once convinced me to do and I have never intended to use it.

You may use this github banner I created, even tho I didn’t finish it yet (wanted to replace the text).

![export-github_1280x640](https://github.com/ocornut/imgui/assets/8225057/9bf98541-559d-4049-94db-7eb96ef83b6e)
![export-banner_2000px](https://github.com/ocornut/imgui/assets/8225057/ede93db6-7b80-462f-bbc9-d3916d46a57f)

**Reply by ghost** [reply]
ok ! so are you saying that dear imgui doesn't have a official logo right now
I needed logo for my website I wanted to add Imgui to my c++ desktop app section and i was searching for something that would match with my other sections and that's why i was searching for a logo
this library is one of the best libraries that i found on internet to use with so I really wanted to add it to my stack

**Reply by ocornut** [reply]
> ok ! so are you saying that dear imgui doesn't have a official logo right now

The image I posted are "official" it's just not yet the quality I want it to be so I'm not showing it much.

**Reply by ghost** [reply]
ok understood ! thankyou for responding for my question , I will use banner ***

**Comment by ocornut** [comment]
Linking to #4009 for ref.
